src/modules/pokemon.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable object-curly-newline */
+/* eslint-disable no-plusplus */
+/* eslint-disable import/prefer-default-export */
+import '../style.css';
+import like from '../assets/like.png';
+import { addLike, catchLike } from './likes.js';
+
+const $list = document.querySelector('.pokemon-list');
+const url = 'https://pokeapi.co/api/v2/pokemon/';
+const array = [];
+
+const fetchPokemon = async (id) => {
+  await fetch(url + id)
+    .then((response) => response.json())
+    .then((data) => {
+      // eslint-disable-next-line no-use-before-define
+      array.push(data);
+    });
+};
+
+export const displayList = (array) => {
+  array.forEach((poke) => {
+    const { name, id, sprites, types } = poke;
+    $list.innerHTML += `
+        <div class="card">
+            <div class="container-sprite"><img src="${sprites.front_default}" alt="${name}"></div>
+            <p class="name">${name}</p>
+            <p class="types">${types.map((type) => type.type.name).join(', ')}</p>
+            <p class="id">#${id}</p>
+            <div class="buttons-container">
+                <button class="button-comments">Comments</button>
```

Comment:
the comment button is not functional , please add this part of work in project requirements you were asked to create a popup windows for adding and displaying comments .

**project requirements :**
When a user clicks on the "Comments" button, the Comments popup appears.
Comments popup
When the popup loads, the webapp retrieves data from:
The selected API and shows details about the selected item.
The Involvement API to show the item comments.
When the user clicks on the "Comment" button, the data is recorded in the Involvement API and the screen is updated.